content/en/docs/setup/minikube.md

```diff
@@ -144,7 +144,7 @@ This brief demo guides you on how to start, use, and delete Minikube locally. Fo
     Stopping "minikube"...
     "minikube" stopped.
     ```
-	For more information, see [Stopping a Cluster](docs/setup/minikube/#stopping-a-cluster).
+	For more information, see [Stopping a Cluster](/docs/setup/minikube/#stopping-a-cluster).
```

Comment:
please replace
/docs/setup/minikube/#stopping-a-cluster
to
minikube.md#stopping-a-cluster